src/encoded/static/components/static-pages/placeholders/MdSortableTable.js

```diff
@@ -0,0 +1,643 @@
+        headerColumnWidths: PropTypes.array,
+    }
+
+    constructor(props) {
+        super(props);
+
+        this.setHeaderWidths = this.setHeaderWidths.bind(this);
+        this.onAdjusterDrag = this.onAdjusterDrag.bind(this);
+
+        this.state = {
+            'widths': (props.headerColumnWidths && props.headerColumnWidths.slice(0)) || null
+        };
+    }
+
+    onClick(index) {
+        this.props.onStateChange.bind(this)(index);
+    }
+
+    setHeaderWidths(idx, evt, r){
+        const { setHeaderWidths } = this.props;
+        const { widths } = this.state;
+        if (typeof setHeaderWidths !== 'function'){
+            throw new Error('props.setHeaderWidths not a function');
+        }
+        setTimeout(()=> setHeaderWidths(widths.slice(0)), 0);
+    }
+
+    onAdjusterDrag(idx, evt, r){
+        this.setState(({ widths }) => {
+            const nextWidths = widths.slice(0);
+            nextWidths[idx] = r.x;
+            return { 'widths': nextWidths };
+        });
+    }
+
+    getWidthFor(idx){
+        const { widths } = this.state;
+        return Array.isArray(widths) && widths[idx];
+    }
+
+    render() {
+        const { columns, sortings, iconAsc, iconDesc, iconBoth } = this.props;
+        const headers = columns.map(((column, index) => {
+            const sorting = sortings[index];
+            return (
+                <SortableTableHeaderItem {...{ sortable: column.sortable, index, header: column.header, sorting, iconAsc, iconDesc, iconBoth }}
+                    key={index}
+                    onClick={this.onClick.bind(this)}
+                    onAdjusterDrag={this.onAdjusterDrag}
+                    setHeaderWidths={this.setHeaderWidths}
+                    width={this.getWidthFor(index)}  />
+            );
+        }).bind(this));
+
+        return (
+            <div className="markdown-table-headers-row">
+                <div className="columns clearfix" style={{ left: 0 }}>
+                    {headers}
+                </div>
+            </div>
+        );
+    }
+}
+
+function SortableTableRow(props) {
+    const { data, columns } = props;
+    const tds = columns.map(function (item, index) {
+        let value = data[item.key];
+        if (item.render) {
+            value = item.render(value);
+        }
+        return (
+            <div className="markdown-table-column-block" style={{ width: item.width }} data-field={item.key} key={item.key}>
+                <div className="inner"><span className="value"><Markdown>{value.toString()}</Markdown></span></div>
+            </div>
+        );
+    }.bind(this));
+
+    return (
+        <div className="markdown-table-row">
+            <div className="columns clearfix result-table-row" draggable="false" style={{ minHeight: '46px' }}>
+                {tds}
+            </div>
+        </div>
+    );
+}
+
+function SortableTableBody(props) {
+    const { data, columns, fullRowWidth } = props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['fullRowWidth' is missing in props validation (react/prop-types)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4682591)